charts/appflowy/charts/appflowy-ai/templates/ingress/nginx/ai.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.global.ingress.nginx.enabled -}}
```

Comment:
I will need to check the AppFlowy source code again to see if /ai is being used at all, even though it is present in the official nginx conf.

Reply:
I just kind of assumed that it might be and replicated the nginx.conf, but from what I've seen it's currently just used via appflowy-cloud which as configured now uses the internal kubernetes service